p2panda-js/jest.config.ts

```diff
@@ -10,6 +10,9 @@ export default {
     wasm: '<rootDir>/wasm',
   },
   modulePathIgnorePatterns: ['<rootDir>/wasm-web'],
+  // Skip reporting coverage for auto-generated wasm module that should be
+  // tested from p2panda-rs.
+  coveragePathIgnorePatterns: ['<rootDir>/wasm'],
```

Comment:
Interesting question actually: What is our testing strategy for `wasm`?

Reply:
This was one of my christmas PRs. We do have tests for the wasm target code in p2panda-rs now: https://github.com/p2panda/p2panda/pull/184

Reply:
We use https://rustwasm.github.io/wasm-bindgen/wasm-bindgen-test/index.html

Reply:
I couldn't figure out how to add the wasm-bindgen-tests to our test coverage though :-(

Reply:
What a meanie...